release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -2508,4 +2508,37 @@ link:https://access.redhat.com/solutions/6246811[{product-title} 4.8.4 container
 [id="ocp-4-8-4-upgrading"]
 ==== Upgrading
 
+To upgrade an existing {product-title} 4.8 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions. CSV. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972478[*BZ#1972478*])
+
+
+[id="ocp-4-8-5"]
+=== RHBA-2021:3121 - {product-title} 4.8.5 bug fix update
+
+Issued: 2021-08-16
+
+{product-title} release 4.8.5 is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2021:3121[RHBA-2021:3121] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:3122[RHBA-2021:3122] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6261051[{product-title} 4.8.5 container image list]
+
+[id="ocp-4-8-5-bug-fixes"]
+==== Bug fixes
+
+* Previously, `oc logs` did not work against `BuildConfig` objects with `JenkinsPipelineStrategy` defined. With this update, `oc logs` now works with pipeline builds. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974267[*BZ#1974267*])
+
+* Previously, when a `Keepalived` container that held VIP was denoted as `SIGTERMed`, no VRRP proactive message was sent. Consequently, the VIP migrated to another node after it timed out. With this update, a `Keepalived` container that holds VIP denoted as `SIGTERMed` sends a `VRRP priority 0` advertisement message. As a result, there is now a faster VIP migration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1620607[*BZ#1620607*])
+
+* Previously, `Kameletbinding` could be used to create `action` and `sink` Kamelets, however, only Kamelets of the `source` type should have been listed. This update removes the option to select `sink` and `action` type Kamelets. As a result, `source` Kamelets are the only type shown in the catalog for event sources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1972258[*BZ#1972258*])
+
+[id="ocp-4-8-5-features"]
+==== Features
+
+[id="ocp-4-8-5-egress-ip-enhancement"]
+===== egress IP enhancement
```

Comment:
I see `egress` is capitalized throughout our repo when starting a sentence/section. So I think capitalizing it would be fine here: `Egress IP address enhancement`